packages/flutter/flutter_rust_bridge.yaml

```diff
@@ -0,0 +1,5 @@
+rust_input: crate
+rust_root: rust/
+dart_output: lib/src/rust
+web: false
+dart_format_line_length: 110
```

Comment:
Not a requirement, but for futureproofing `full_dep` can be enabled to fully generate C headers.

```
full_dep: true
c_output: ios/Classes/breez_sdk_spark_flutter.h
duplicated_c_output: [macos/Classes/breez_sdk_spark_flutter.h]
```

Generated Dart entrypoint class name can also be defined here
```
dart_entrypoint_class_name: BreezSdkSpark
```
to avoid conflicts with libraries that use the default `RustLib` class name. 

This change will effect 
- [packages/flutter/lib/breez_sdk_spark.dartL#3](https://github.com/breez/spark-sdk/pull/204/files#diff-c0b99ff4181a420816720f055db1a81cab593698207c92746f8fdd65ad67e2f4R3)
- [packages/flutter/README.mdL#9](https://github.com/breez/spark-sdk/pull/204/files#diff-4fcbcf25c28836c707a626da4ab29755eedfbdf9a7586c0656a2d25aeb7c0d78R9)